book/concepts.md

```diff
@@ -1,6 +1,131 @@
 # Concepts
+Hydroflow is different from other distributed systems infrastructure, so you probably have questions, like:
+What is Hydroflow? How does it work? What is special about it, and what are the key concepts to understand?
+
+This chapter covers those basic questions. We start simple describing Hydroflow, and build up to an understanding of 
+what makes Hydroflow uniquely powerful.
+
+But in case you want a preview of the Hydroflow goodies, here are the main themes:
+1. **Distributed Correctness**: Hydroflow's type system can prevent distributed system bugs at compile time. (One example: will your code
+produce the same results if you deploy on a single machine or replicate on a distributed cluster of machines?)
+2. **Local Efficiency**: Hydroflow compiles your dataflow programs into efficient, low-latency, single-threaded executables.
```

Comment:
This sounds like a bit of the **Hydro** stack as well?

Reply:
Hmm ... true even in the narrower scope of hydroflow so I think we should highlight.